libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpMp4aReader.java

```diff
@@ -70,16 +77,24 @@ public void createTracks(ExtractorOutput extractorOutput, int trackId) {
   public void onReceivingFirstPacket(long timestamp, int sequenceNumber) {
     checkState(firstReceivedTimestamp == C.TIME_UNSET);
     firstReceivedTimestamp = timestamp;
-  }
+    try {
+      numSubFrames = getNumOfSubframesFromMpeg4AudioConfig(payloadFormat.fmtpParameters);
```

Comment:
why not putting this in the constructor? it doesn't seem that f`fmtpParameters` can change in between packets. this way `numSubFrames` can be final

Reply:
although it's not ideal, you can throw a IAE wrapping the parserException.

Make it's caught in `RtspMediaPeriod.onLoadError()`